data/category-ir-bourse

```diff
@@ -0,0 +1,41 @@
+# Iranian stock exchange websites which will not open without an Iranian IP
+# Use to make them direct connections so they open inside Iran
```

Comment:
this comment does not necessarily have to be introduced in every sub-category.

Reply:
Removed